crates/gpui/src/platform/keystroke.rs

```diff
@@ -155,12 +155,13 @@ impl std::fmt::Display for Keystroke {
         }
         let key = match self.key.as_str() {
             "backspace" => '⌫',
-            "up" => '↑',
-            "down" => '↓',
-            "left" => '←',
-            "right" => '→',
+            "up" => '▲',
+            "down" => '▼',
+            "left" => '◀',
+            "right" => '▶',
             "tab" => '⇥',
             "escape" => '⎋',
+            "delete" => '⌦',
```

Comment:
I have test to add the `enter` keyboard symbol (↩ or ⏎), but that in `Zed Sans` render is not good.